Source/WebKit/UIProcess/PDF/WKPDFPageNumberIndicator.mm

```diff
@@ -165,8 +165,10 @@ - (void)show
 
 - (void)hide:(NSTimer *)timer
 {
+    static constexpr auto effectivelyTransparentAlpha = 0.0101;
     auto animations = [view = retainPtr(self)] {
-        [view setAlpha:0];
+        // FIXME: <rdar://162795344> Remove this workaround and directly setAlpha:0 after rdar://154649008.
```

Comment:
Should we go all the way to zero in the animation completion handler?

Reply:
Agreed, I changed it in https://github.com/WebKit/WebKit/commit/20a0b2beef2a3aa89ce1db404add6c9a690af0ef